src/stories/elements/lift.stories.js

```diff
@@ -0,0 +1,115 @@
+import React from 'react';
+import Lift from '../../components/schedule-visualizer/lift/lift';
+import { liftStyles } from '../../components/schedule-visualizer/lift/liftContainer';
+
+export default {
+  title: 'Lift',
+  component: Lift,
+};
+
+export const LiftState = () => {
+  const classes = liftStyles();
+  const liftBasic = {
+    x: 0,
+    y: 0,
+    width: 100,
+    height: 100,
+    rx: 10,
+    ry: 10,
+  };
+  const liftEmergency = { ...liftBasic, style: classes.emergency };
+  const liftFire = { ...liftBasic, style: classes.fire };
+  const liftOffLine = { ...liftBasic, style: classes.offLine };
+  const liftHuman = { ...liftBasic, style: classes.humanMode };
+  const liftUnknown = { ...liftBasic, style: classes.liftOnAnotherFloor };
+
+  return (
+    <>
+      <h2>Emergency</h2>
+      <svg width="100" height="100">
+        <Lift
+          lift={liftEmergency}
+          liftMotionText={{
+            x: 50,
+            y: 50,
+            text: 'L1->L2',
+            fontSize: '0.8rem',
+          }}
+          liftModeText={{
+            x: 50,
+            y: 25,
+            text: 'EMERGENCY!',
+            fontSize: '0.8rem',
```

Comment:
These are using fixed values for `liftMotionText` and `liftModeText` props, they may diverge from the actually style used in the app. Is it possible to refactor this to render the lift with a `Lift` and `LiftState` object, letting the `LiftContainer` component handle the props?

Reply:
Yeah, I understand what are you saying. I guess I can refactor the lift component and the storybook so we can get close to the current style without much effort (e.g: using the mock data).

The liftContainer has calculations used to scale the components to the canvas that we have on Romi-dashboard, so we can't use the liftContainer just to handle the styles. But I guess we can separate the style and text handler from the Component and call it here.

We will have as many states as lifts.

Reply:
What do you think of this?:

* We can use the Storybook to visualize (in this case) the lift components and its states and have a playground to modify the visual component.

* And the calculations and states of the container components should be tested with jest/enzyme.

Reply:
I think an advantage of storybook is If we can get the storybook rendered components as close as possible to the actual render code used in the actual app, it can help visualize how different calculations affect the look.

Of course its still would be good to have tests in jest. Storybook would help with the visual design aspects, and jest would help in the technical aspects.